2020-10-27 02:10:12.183 - [Info] MITM Version: 1.26
2020-10-27 02:10:12.189 - [Info] Auto Access Enabled: true
2020-10-27 02:10:12.190 - [Debug] [Init] Auto Access Barrier: true
2020-10-27 02:10:12.211 - [Info] SSH man-in-the-middle server for 172.20.0.203 listening on 0.0.0.0:10002
2020-10-27 02:22:09.045 - [Debug] [Connection] Attacker connected: 193.105.134.45 | Client Identification: SSH-2.0-paramiko_1.16.1
2020-10-27 02:22:10.633 - [Debug] [Auth] Attacker 193.105.134.45 trying to authenticate with "none"
2020-10-27 02:22:10.633 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2020-10-27 02:22:10.913 - [Debug] [Auth] Attacker 193.105.134.45 trying to authenticate with "password"
2020-10-27 02:22:10.914 - [Debug] [Auto Access] Attacker: 193.105.134.45, Threshold: 1, Attempts: 1
2020-10-27 02:22:10.914 - [Debug] [Auto Access] Compromising the honeypot
2020-10-27 02:22:10.914 - [Debug] [Auto Access] Adding the following credentials: "22:admin"
2020-10-27 02:22:11.335 - [Debug] [Auth] Valid credentials - Password Authentication
2020-10-27 02:22:11.336 - [Debug] [LXC] Attempting to connect to the honeypot: 172.20.0.203
2020-10-27 02:22:23.071 - [Debug] [Connection] Attacker connected: 141.98.81.141 | Client Identification: SSH-2.0-Go
2020-10-27 02:22:26.343 - [Debug] [Auth] Attacker 141.98.81.141 trying to authenticate with "none"
2020-10-27 02:22:26.344 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2020-10-27 02:22:26.513 - [Debug] [Auth] Attacker 141.98.81.141 trying to authenticate with "password"
2020-10-27 02:22:26.514 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2020-10-27 02:22:26.516 - [Debug] [Auth] Attacker: 141.98.81.141 has so far made 1 attempts. Remaining: 5 attempts
2020-10-27 02:22:26.730 - [Debug] [Connection] Attacker closed the connection
2020-10-27 02:22:26.738 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2020-10-27 02:22:27.859 - [Debug] [Connection] Attacker closed connection
2020-10-27 02:22:28.329 - [Debug] [LXC] Container's OpenSSH server ended connection
2020-10-27 02:22:28.330 - [Debug] [LXC] Container's OpenSSH server closed connection
2020-10-27 02:23:46.502 - [Debug] [Connection] Attacker connected: 193.105.134.45 | Client Identification: SSH-2.0-PuTTY_Release_0.64
2020-10-27 02:23:47.973 - [Debug] [Auth] Attacker 193.105.134.45 trying to authenticate with "none"
2020-10-27 02:23:47.974 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2020-10-27 02:23:48.253 - [Debug] [Auth] Attacker 193.105.134.45 trying to authenticate with "password"
2020-10-27 02:23:48.254 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2020-10-27 02:23:48.255 - [Debug] [Auth] Attacker: 193.105.134.45 has so far made 1 attempts. Remaining: 5 attempts
2020-10-27 02:23:48.543 - [Debug] [Connection] Attacker closed the connection
2020-10-27 02:23:52.912 - [Debug] [Connection] Attacker connected: 195.3.147.47 | Client Identification: SSH-2.0-PuTTY_Release_0.59
2020-10-27 02:23:54.401 - [Debug] [Auth] Attacker 195.3.147.47 trying to authenticate with "none"
2020-10-27 02:23:54.401 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2020-10-27 02:23:54.672 - [Debug] [Auth] Attacker 195.3.147.47 trying to authenticate with "password"
2020-10-27 02:23:54.673 - [Debug] [Auth] Valid credentials - Password Authentication
2020-10-27 02:23:54.673 - [Debug] [LXC] Attempting to connect to the honeypot: 172.20.0.203
2020-10-27 02:23:54.722 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2020-10-27 02:23:56.613 - [Debug] [Connection] Attacker closed connection
2020-10-27 02:23:56.625 - [Debug] [LXC] Container's OpenSSH server ended connection
2020-10-27 02:23:56.625 - [Debug] [LXC] Container's OpenSSH server closed connection
